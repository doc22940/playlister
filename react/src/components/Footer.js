import React, { Component }  from 'react';

import poweredByDarkSky from '../media/poweredby-darksky-darkbackground.png';

class Footer extends Component {
  closeMenu() {
    try {
      document.getElementsByClassName('navbar-toggler')[0].click();
    } catch (err) {
      console.error('trying to close the navbar on mobile. Ignore.', err);
    }
  }

  render() {
    return (
      <footer className='footer bg-dark'>
        <div className='text-center'>
          <span className='h3 text-light align-middle'>
            USED TO BE...
          </span>
          <a
            href='https://darksky.net/poweredby/'
            target='_blank'
            rel='noopener noreferrer'
          >
            <img
              alt='Powered By DarkSky'
              src={poweredByDarkSky}
              style={{ width: '50%' }}
            />
          </a>
        </div>
      </footer>
    );
  }
}

export default Footer;

