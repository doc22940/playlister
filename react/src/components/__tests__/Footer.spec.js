import React from 'react';
import renderer from 'react-test-renderer';

import Footer from '../Footer.js';

test('Footer snapshot', () => {
  const component = renderer.create(
    <Footer />,
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
