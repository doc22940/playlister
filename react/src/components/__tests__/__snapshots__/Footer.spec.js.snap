// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot 1`] = `
<footer
  className="footer fixed-footer bg-primary p-3 p-3"
>
  <div
    className="text-center"
  >
    <span
      className="h6 text-light align-middle justify-content-center"
    >
      Built by rad people at Northview Church!
    </span>
  </div>
</footer>
`;
